.nvmrc

```diff
@@ -1 +1 @@
-10.10.0
\ No newline at end of file
+12.14.1
```

Comment:
Prevents netlify build from failing, error:

`error @typescript-eslint/eslint-plugin@2.17.0: The engine "node" is incompatible with this module. Expected version "^8.10.0 || ^10.13.0 || >=11.10.1". Got "10.10.0"`

Do you think this version is good? It is the current LTS version.